features/nanostack/nanostack-hal-mbed-cmsis-rtos/arm_hal_fhss_timer.cpp

```diff
@@ -114,7 +111,7 @@ static int platform_fhss_timer_start(uint32_t slots, void (*callback)(const fhss
         equeue = mbed_highprio_event_queue();
         MBED_ASSERT(equeue != NULL);
 #endif
-        timer->start();
+        sleep_manager_lock_deep_sleep();
```

Comment:
This should really be `HighResClock::lock()`, self-documenting why it's needed - keeping that clock running for `platform_fhss_timeout_read`.

Reply:
updating this PR or this will be in the new PR?

Reply:
Oh, let's do that now.